ansible/files/creategraph.py

```diff
@@ -63,7 +63,8 @@ def generate_dpg(self):
         for dev in self.devices:
             hostname = dev.get('Hostname', '')
             managementip = dev.get('ManagementIp', '')
-            if hostname and 'fanout' in dev['Type'].lower():
+            devtype = dev['Type'].lower()
+            if hostname and ('fanout' in devtype or 'ixiachassis' in devtype):
```

Comment:
Just a small question - why `ixiachassis` as opposed to just `ixia`?

Reply:
Or `ixia_tester` or something?

Reply:
My understanding from Wei is that it is a chassis with line cards.